### PR TITLE
Add compact preset

### PR DESCRIPTION
**Pull request by tomivirkki**
Connects to https://github.com/vaadin/vaadin-core/issues/212

---
This change is [Reviewable](https://reviewable.io/reviews/vaadin/vaadin-lumo-styles/58)

### CODE REVIEW

File: demo/sizing-and-spacing.html [hunk 1]
```diff
@@ -244,6 +244,22 @@ <h3>Tall</h3>
       </template>
     </nice-demo-snippet>
 
+    <h2 demo-section>Compact preset</h2>
+    <p>The "compact" preset is a set of predefined sizing/spacing properties that reduce the overall application size to better fit a large number of content on the screen.</p>
```
Comment (jouni) [inline demo/sizing-and-spacing.html]: Clarify documentation and fix grammar:

“The compact preset defines values for the sizing and spacing properties to reduce the visual space required by components to better fit a large amount of content on the screen.”

File: mixins/overlay.html [hunk 2]
```diff
@@ -12,7 +12,7 @@
         bottom: var(--lumo-space-m);
         left: var(--lumo-space-m);
         /* Workaround for Edge issue (only on Surface), where an overflowing vaadin-list-box inside vaadin-select-overlay makes the overlay transparent */
-        outline: 0px solid transparent;
+        outline: 0 solid transparent;
```
Comment (jouni) [inline mixins/overlay.html]: Are you sure this doesn’t create a regression? I’m not certain Edge didn’t care about the unit here…

File: test/sink.html [hunk 3]
```diff
@@ -5,6 +5,8 @@
     <title>Lumo components kitchen sink test</title>
     <meta name="viewport" content="width=device-width">
     <script src="../../webcomponentsjs/webcomponents-loader.js"></script>
+    <!-- <link rel="import" href="../presets/compact.html"> -->
+
     <link rel="import" href="all-imports.html">
     <!-- <link rel="import" href="compact-theme.html"> -->
```
Comment (jouni) [inline test/sink.html]: Nit: could be removed

File: demo/sizing-and-spacing.html (hunk 1 above)
Comment (tomivirkki) [reply demo/sizing-and-spacing.html]: Done.

File: mixins/overlay.html (hunk 2 above)
Comment (tomivirkki) [reply mixins/overlay.html]: Good point, better play it safe. Added a linter ignore instead.

File: test/sink.html (hunk 3 above)
Comment (tomivirkki) [reply test/sink.html]: Done.

**Pull request merged**